gazebo/rendering/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ set (sources
   COMVisual.cc
   ContactVisual.cc
   Conversions.cc
+  CustomAutoParamDataSource.cc
```

Comment:
we can conditionally add the `.cc` file to `sources` and `.hh` file to `headers` based on the `OGRE_VERSION` like how the `deferred_shading` folder is handled [later in this file](https://github.com/osrf/gazebo/blob/gazebo11_11.4.0/gazebo/rendering/CMakeLists.txt#L144-L146)

Reply:
thanks the windows build is fixed now

[![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gazebo-ci-pr_any-windows7-amd64&build=3169)](https://build.osrfoundation.org/job/gazebo-ci-pr_any-windows7-amd64/3169/) https://build.osrfoundation.org/job/gazebo-ci-pr_any-windows7-amd64/3169/